### ISSUE TITLE
Translation on website

### ISSUE THREAD

**Issue by komachi** [ProjectMeshnet/Hyperboria#3]
It's not working now. You must intall gettext and there may be some problems with includes, so you must edit config of your webserver and disable multiple processes.

**Comment by dansup** [comment]
Yes, it appears I will have to re-compile php with gettext support. I believe this is a small price to pay for future localization, and I'd like to thank you again for implementing this.

**Comment by dansup** [comment]
I recently rewrote the site from scratch, will be implementing localization again and I will use any strings I can from your translations.

**Comment by dansup** [comment]
We now have four language translations besides English! I'm working on fixing gettext on my server, then I will add a dropdown menu to set desired languages. @komachi thanks again for bringing localization to this, we really need it now.
